uber/config.py

```diff
@@ -352,6 +352,14 @@ def PAGE_PATH(self):
     def PAGE(self):
         return cherrypy.request.path_info.split('/')[-1]
 
+    @property
+    def IS_DEALER_REG_REQUEST(self):
+        is_dealer_referer = os.path.basename(cherrypy.request.headers.get('Referer', '')) in ('dealer_registration', 'post_form')
+        is_dealer_post = c.HTTP_METHOD == 'POST' and c.PAGE_PATH == '/preregistration/post_form' and is_dealer_referer
```

Comment:
Hmm. This is fine and I expect it works, but it looks like once we do open real pre-reg, this property will return `true` for folks who've submitted a prereg form after submitting a prereg form. Right now that's not a problem, but I can definitely see us running into problems later (particularly if one year we decide we want to do the inverse of what we did this year).

I'd be a little more comfortable if we could, instead of inspecting the Referer, inspect the parameters for the PSEUDO_DEALER_BADGE type, which would _not_ get passed in via the prereg form unless someone did some monkeying with the badge_type input. I don't know how feasible that'd be in a property, but it should prevent false positives.